backend/baseclass.pm

```diff
@@ -568,6 +568,7 @@ sub reset_consoles {
     # we iterate through all consoles
     for my $console (keys %{$testapi::distri->{consoles}}) {
         #next if ($console eq 'x3270');
```

Comment:
I think this line is now obsolete and can be deleted. It's a very unobvious "TODO" for what you are actually doing in this PR ;)

Reply:
done